lib/appeals_status/service.rb

```diff
@@ -31,8 +46,45 @@ def try_mocks(mock_responses_method, user, response_class)
       nil
     end
 
+    def format_response_reasonably(response)
+      response = response.deep_symbolize_keys
+      format_hearings_reasonably(response)
+      response[:data].each do |appeal|
+        appeal[:events] = appeal[:attributes].delete(:events)
+        appeal[:attributes].each do |k, v|
+          appeal[k] = v
+        end
+        appeal.delete(:attributes)
+        appeal.delete(:relationships)
+      end
+      response[:data]
+    end
```

Comment:
I wonder if you could do this in a custom middleware instead? It's fine here, but I think the if I recall correctly the snake_case middleware should have already symbolized keys. With chained middlewares it might be easier to follow that the purpose of this method is to transform the data so that it can be used to deserialize data elsewhere? 

Or is this maybe because the MockService doesn't require faraday?

NITPICK.